topsort-endpoints.yml

```diff
@@ -24,7 +24,7 @@ security:
   - BearerAuth: []
 
 paths:
-  /auctions:
+  /auctions/:
```

Comment:
Dont' add a trailing slash, we already merged a PR in which the url with a trailing slash will return a 404